Gemfile.lock

```diff
@@ -1,3 +1,10 @@
+PATH
+  remote: /Users/dlhaines/dev/GITHUB/dlh-umich.edu/FORKS/google_groups_gem
```

Comment:
Make sure before final check-in that there's no mention of your local pathnames.